### ISSUE TITLE
compilation errors in "export" sample app

### ISSUE THREAD

**Issue by abstratt** [mendix/mendixplatformsdk-examples#6]
```
$ tsc
script.ts(83,45): error TS2339: Property 'DeprecatedCreateAction' does not exist on type 'typeof microflows'.
script.ts(84,32): error TS2339: Property 'outputVariableName' does not exist on type 'MicroflowAction'.
script.ts(84,72): error TS2339: Property 'entityQualifiedName' does not exist on type 'MicroflowAction'.
```

Apparently code is successfully generated and I can run it (so are those warnings?).

**Comment by reinouts** [comment]
Proposed fix is in PR #7

**Comment by reinouts** [comment]
Solved in release 1.0.2
